_data/navi.yml

```diff
@@ -28,13 +28,26 @@ toc:
           url: constraints.html
         - title: Attributes
           url: attributes.html
-        - title: JSON Beispiel
-          url: json.html
         - title: XML Beispiel
           url: xmlbeispiel.html
         - title: XSD
           url: xsdschema.html
-
+  - title: Beispiele
+    path: /examples/
+    url: examples.html
+    subnav:
+      - title: Reine Metadatenanzeige
+        url: metadataonly.html
+      - title: Authentifizierung nötig
+        url: authentification.html
+      - title: Abschluss eines Nutzungsvertrags
+        url: agreement.html
+      - title: IP-Adressbereichs
+        url: location.html
+      - title: Ansicht, aber keine Speicherungs-/Druckmöglichkeit
```

Comment:
@Duchatelet Ich hab versucht die Beispiele in der Seitennavigation griffiger zu formulieren. Passt das aus deiner Sicht?